strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/security/SecurityTokenProvider.java

```diff
@@ -63,13 +63,20 @@ public void init(@Value("${strongbox.security.jwtSecret:secret}") String secret)
      */
     public String getToken(String subject,
                            Map<String, String> claimMap,
-                           Integer expireSeconds)
+                           Integer expireSeconds, NumericDate issuedAt)
```

Comment:
Cosmetic: Missing `@param issuedAt` :)

Reply:
@Twb3 : 
Cosmetic: Could you please make it so that there's one parameter per line, in order to follow our 
[coding convention](https://strongbox.github.io/developer-guide/coding-convention.html)? Thanks! :)